pkg/reconciler/v1alpha1/configuration/resources/revision_test.go

```diff
@@ -247,5 +247,15 @@ func TestMakeRevisions(t *testing.T) {
 				t.Errorf("MakeRevision (-want, +got) = %v", diff)
 			}
 		})
+
+		t.Run(test.name+"(template)", func(t *testing.T) {
```

Comment:
please don't name tests with `()`, it's so hard to put them on the `-run=X` afterwards